nestkernel/event.h

```diff
@@ -952,6 +952,12 @@ class GapJunctionEvent : public SecondaryEvent
     supported_syn_ids_.push_back( synid );
   }
 
+  static void
+  set_coeff_length( const size_t coeff_length )
+  {
+    coeff_length_ = coeff_length;
```

Comment:
@janhahne I think
```C++
#ifdef _OPENMP
  assert( omp_get_num_threads(void) == 1 );
#endif
```
should make sure this function breaks if it is ever called from a thread-parallel context (see
https://computing.llnl.gov/tutorials/openMP/#OMP_GET_NUM_THREADS).